hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/HapiSpecRegistry.java

```diff
@@ -535,6 +536,11 @@ public void saveAccountId(String name, AccountID id) {
         put(asAccountString(id), name);
     }
 
+    public void saveNodeId(String name, long nodeId) {
+        put(name, nodeId);
+        put(String.valueOf(nodeId), name);
+    }
```

Comment:
Shouldn't we save like `saveNodeId(AccountID name, long nodeId)`. I am not sure if this works

Reply:
The name is a convenient identifier for hapi tests.  It works now.